api/gridStationSchema.js

```diff
@@ -0,0 +1,51 @@
+//task 2 /create schema for gridStation
+
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const requiredString = {
+    type: String,
+    required: true
+};
+
+const requiredNumber = {
+    type: Number,
+    required: true
+};
+
+const userSchema = new Schema({
+    firstName: requiredString,
+    lastName: requiredString,
+    userName: requiredString,
+    email: {
+        type: String,
+        required: [true, "can't be blank"],
+        match: [/\S+@\S+\.\S+/, 'is invalid'],
+        index: true
+    },
+    phoneNumber: Number,
+    StationAddress: {
+        latitude: {
+            ...requiredNumber,
+            min: -90,
+            max: 90,
+        },
+        longitude: {
+            ...requiredNumber,
+            min: -180,
+            max: 180
+        }
+    },
+    StationNumber: requiredNumber,
+    No_of_chargingPoint= requiredNumber,
```

Comment:
always follow `camelCase` convention for js code. Change the snakeCase to camel case.
change this to registration number or something else. If that's what it is.